### PR TITLE
Added google and a guide to ask good questions.

### PR DESCRIPTION
**Pull request by harshhh-dev**
Added google and a guide to ask good questions.

### CODE REVIEW

**Comment by lenscas** [comment]
you haven't added them to the available commands yet. Look in commands.rs on how the others are done

File: src/commands/google.rs [hunk 1]
```diff
@@ -1 +1,26 @@
...
+};
 
+#[command]
+#[aliases("browser", "google")]
+#[description("Teaches users how to paste code neatly")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output"]
+pub(crate) async fn codeblock(ctx: &Context, msg: &Message) -> CommandResult {
```
Comment (lenscas) [inline src/commands/google.rs:15]: you probably want to look at the codeblock's command. As that one has a bigger delay and can run from any channel

**Review by lenscas — CHANGES_REQUESTED** [review]

File: src/commands/google.rs (hunk 1 above)
Comment (harshhh-dev) [reply src/commands/google.rs:15]: done.

Comment (lenscas) [reply src/commands/google.rs:15]: I think you need to push,as I don't see the changes yet

File: src/commands/google.rs [hunk 2]
```diff
@@ -1 +1,23 @@
...
+};
 
+#[command]
+#[aliases("browser", "google")]
+#[description("Reminds users that google exists")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output_ever_channel"]
+pub(crate) async fn codeblock(ctx: &Context, msg: &Message) -> CommandResult {
```
Comment (lenscas) [inline src/commands/google.rs:15]: this should be google, not codeblock

File: src/commands/questions.rs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+};
+
+#[command]
+#[aliases("paste", "code")]
+#[description("Teaches users how to paste code neatly")]
+#[usage("")]
+#[example = ""]
+#[help_available]
+#[bucket = "potentially_big_output_ever_channel"]
+pub(crate) async fn codeblock(ctx: &Context, msg: &Message) -> CommandResult {
```
Comment (lenscas) [inline src/commands/questions.rs:15]: this should be questions, not codeblock

**Review by lenscas — CHANGES_REQUESTED** [review]

File: src/commands/questions.rs (hunk 3 above)
Comment (harshhh-dev) [reply src/commands/questions.rs]: k fixed that for both google and questions

File: src/commands/google.rs (hunk 2 above)
Comment (harshhh-dev) [reply src/commands/google.rs]: fixed

**Review by lenscas — APPROVED** [review]

**Pull request merged**